charts/bandstand-cron-job/templates/pv.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if and .Values.s3Volume (default false .Values.s3Volume.enabled) }}
+{{- $relName := .Release.Name -}}
+{{- if .Values.s3Volume.nameSuffix }}
+{{- $relName = print .Release.Name "-" .Values.s3Volume.nameSuffix }}
+{{- end }}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ .Values.s3Volume.pvName | default (print $relName "-pv") }}
+  labels:
+    {{- include "bandstand-cron-job.labels" . | nindent 4 }}
+spec:
+  capacity:
+    storage: {{ .Values.s3Volume.persistent | default "5Gi" }}
+  volumeMode: Filesystem
+  accessModes:
+    - ReadOnlyMany
+  persistentVolumeReclaimPolicy: Retain
+  csi:
+    driver: s3.csi.aws.com
+    volumeHandle: {{ .Values.s3Volume.volumeHandle | default (print "s3-" $relName "-volume") }}
+    volumeAttributes:
+      bucketName: {{ .Values.s3Volume.bucketName }}
+      authenticationSource: {{ .Values.s3Volume.authenticationSource | default "pod" }}
```

Comment:
I don't think we should make `authenticationSource` configurable - it should always be `pod`